### ISSUE TITLE
`[aria-current]` not working with Tailwind CSS

### ISSUE THREAD

**Issue by vietthedev** [denoland/fresh#2123]
I've just updated to Fresh 1.6 and migrated to Tailwind. I followed the docs after #2019 to style active links with `[aria-current]` but it doesn't work while `aria-[current='page']` works well.

**Comment by marvinhagemeister** [comment]
Thanks for sharing. That's a good find and we should document the differences. Opened #2137 for that.

**Comment by Sidewinder1138** [comment]
It seems to me that the "Styling active links" example page needs an update regarding this.
https://fresh.deno.dev/docs/examples/active-links

The suggestion in the "Tailwind" section doesn't work... but the syntax suggested in this issue does... (I'm using latest Fresh with proper Tailwind).

**Comment by marvinhagemeister** [comment]
Resolved by https://github.com/denoland/fresh/pull/2417
